ui/app/src/components/Navigation/PathNavigator/Widget.tsx

```diff
@@ -389,9 +401,29 @@ export function WidgetUI(props: any) {
   );
 
   return (
-    <section className={clsx(classes.root, props.classes?.root, state?.collapsed && 'collapsed')}>
+    <section
+      className={clsx(
+        classes.root,
+        props.classes?.root,
+        state?.collapsed && 'collapsed',
+        container.baseClass,
+        state?.collapsed ? container.collapsedClass : container.expandedClass
+      )}
+      style={{
+        ...container.baseStyle,
+        ...(state?.collapsed ? container.collapsedStyle : container.expandedStyle)
+      }}
+    >
       <Header
-        icon={icon}
+        icon={
+          icon.baseClass
+            ? `${icon.baseClass} ${state?.collapsed ? icon.collapsedClass : icon.expandedClass}`
+            : icon
+        }
+        style={{
+          ...icon.baseStyle,
+          ...(state?.collapsed ? icon.collapsedStyle : icon.expandedStyle)
+        }}
```

Comment:
- Is state ever undefined here? Perhaps we need a wrapping loading state or resource?
- If state is ever undefined, these conditions would always use the expanded style, which I'm not sure that's right

Reply:
state is never undefined, this was my mistake. I will remove state validation.